web/src/beta/features/Editor/tabs/publish/Nav/PublishModal/index.tsx

```diff
@@ -174,7 +174,7 @@ const PublishModal: React.FC<Props> = ({
           <HideableSection showOptions={showOptions}>
             <Wrapper>
               <Subtitle size="body">{t("Search engine indexing")}</Subtitle>
-              <ToggleButton checked={searchIndex} onChange={handleSearchIndexChange} />
+              <Toggle checked={searchIndex} onChange={handleSearchIndexChange} />
```

Comment:
I think this toggle should be the field/Toggle. The design is as shown in the screenshot 🙏🏻 
![Uploading Screenshot 2023-08-31 at 10.24.14.png…]()

Reply:
Of course don't set the description as `Description around`. That seems to be just boiler text since maybe @lavalse didn't have time to think what the descriptions should be in many places. We can add those later easily enough 👍🏻

Reply:
![image](https://github.com/reearth/reearth/assets/20891087/cf1e4a2a-13d4-4f6a-9595-5ce9864f9886)
It's in the publish page so it looks like this. Let me know if this design is changed, I kept it similar to as before.